### ISSUE TITLE
bug(developer): kmc not finding build files in .kps on Linux

### ISSUE THREAD

**Issue by darcywong00** [keymanapp/keyman#10942]
On my Linux dev, I can build master branch e.g. `./build.sh release/sil/sil_euro_latin` which uses kmcomp.

But on the `staging-17.0` branch, the package build fails (using kmc)  on `./build.sh -k release/sil/sil_euro_latin`.
```
sil_euro_latin.kps - error KM04008: Keyboard sil_euro_latin was listed in <Keyboards> 
but a corresponding .kmx file was not found in <Files>
```

If I change the slashes in sil_euro_latin.kps to
```
<Name>../build/sil_euro_latin.kmx</Name>
```
it builds. I guess this is a subtle different from kmcomp and kmc?

Maybe belongs in main repo

**Comment by mcdurdin** [comment]
Yes, sounds like an issue with kmc. Puzzled though, because it was all building when I was testing earlier on Linux, so feels like a regression. Looking at it.

**Comment by mcdurdin** [comment]
Note that the version on keyboards/staging-17.0 is very old -- 17.0.194-alpha, because refresh of that branch is still waiting on me to get to it. (There's a lot to it, in particular the merge from master to staging-17.0 due to .keyboard_info metadata moving into .kps files.)  I probably need an issue to track that, perhaps will repurpose this if it turns out to be the issue.

**Comment by mcdurdin** [comment]
Fixed by #10950, will not be used by keyboards staging-17.0 branch until its package.json is refreshed to latest version.
